dnf-json

```diff
@@ -295,169 +165,136 @@ class Solver():
                 )
             })
 
+        return dependencies
+
+
+def setup_cachedir(request):
+    arch = request["arch"]
+    # If dnf-json is run as a service, we don't want users to be able to set the cache
```

Comment:
Do we still support running as a service?

Reply:
No (not yet)